discourse_migration/userguide.md

```diff
@@ -0,0 +1,155 @@
+  ([tested](https://llvm.discourse.group/t/email-interaction-with-discourse/3306/4) on GMail).
+* [Quoting previous topics in an reply](https://meta.discourse.org/t/single-quote-block-dropped-in-email-reply/144802)
+* **TODO:** Creating new topics via email is
+  [supported](https://meta.discourse.org/t/start-a-new-topic-via-email/62977)
+  but not configured at the moment. We would need to set up an email address
+  per category and give Discourse POP3 access to that email account. This sounds
+  like a solvable issue.
+* You can filter incoming emails in your email client by category using the
+  `List-ID` email header field.
+
+## Mapping of mailing lists to categories
+
+This table explains the mapping from mailing lists to categories in Discourse.
+The email addresses of these categories will remain the same, after the
+migration.
+
+| Mailing lists         | category in Discourse |
+|-----------------------|--------------------|
+| All-commits           | no migration at the moment |
+| Bugs-admin            | no migration at the moment |
+| cfe-commits           | no migration at the moment |
+| cfe-dev               | llvm-project/clang |
+| cfe-users             | **TODO** llvm-project/clang or create new category? |
+| clangd-dev            | llvm-project/clangd |
+| devmtg-organizers     | **TODO** create new category |
+| Docs                  | infrastructure/documentation |
+| eurollvm-organizers   | **TODO** create new category |
+| flang-commits         | no migration at the moment |
+| flang-dev             | llvm-project/flang |
+| gsoc                  | community/gsoc |
+| libc-commits          | no migration at the moment |
+| libc-dev              | llvm-project/libc |
+| Libclc-dev            | **TODO** create new category |
+| libcxx-bugs           | no migration at the moment |
+| libcxx-commits        | no migration at the moment |
+| libcxx-dev            | llvm-project/libcxx |
+| lldb-commits          | no migration at the moment |
+| lldb-dev              | llvm-project/lldb |
+| llvm-admin            | **TODO** create new category |
+| llvm-announce         | **TODO** create new category |
+| llvm-branch-commits   | no migration at the moment |
+| llvm-bugs             | no migration at the moment |
+| llvm-commits          | no migration at the moment |
+| llvm-dev              | llvm-project/llvm |
+| llvm-devmeeting       | community/devmtg |
+| llvm-foundation       | **TODO**  create new category|
+| Mlir-commits          | no migration at the moment |
+| Openmp-commits        | no migration at the moment |
+| Openmp-dev            | llvm-project/openmp |
+| Parallel_libs-commits | no migration at the moment |
+| Parallel_libs-dev     | **TODO** use llvm-project/pstl or create new category ? |
+| Release-testers       | **TODO** create new category |
+| Test-list             | **TODO** create new category |
+| vmkit-commits         | no migration at the moment |
+| WiCT                  | **TODO** |
```

Comment:
This exists already: https://llvm.discourse.group/c/community/women-in-compilers-and-tools/18

Reply:
thx, I updated the document in main branch